src/openfermion/ops/_fermion_operator.py

```diff
@@ -15,9 +15,10 @@
 from openfermion.config import *
 from future.utils import iteritems
 import numpy
+from _symbolic_operator import *
```

Comment:
You will need to write from ._symbolic_operator import *. We tend to discourage the "import *" thing though - better if you can write explicitly what you are importing.

Reply:
`from ._symbolic_operator import SymbolicOperator, SymbolicOperatorError` should work I think?

Reply:
I think so, yes